java-manta-client-unshaded/src/main/java/com/joyent/manta/exception/MantaErrorCode.java

```diff
@@ -180,4 +522,31 @@ public static MantaErrorCode valueOfCode(final Object object) {
 
         return valueOfCode(object.toString());
     }
+
+    /**
+     * Populates an Inclusive Error Map for {@link MantaErrorCode} by its code value.
+     *
+     * @return a thread-safe Manta Errors HashMap.
+     * @throws MantaClientException if a thread-safe access is not granted for Error Map.
+     */
+    public static Map<String, MantaErrorCode> createErrorMap() throws MantaClientException {
+        errorMap = Collections.synchronizedMap(new HashMap<>(values().length));
```

Comment:
Why are you assigning the map here in the method that is returning a reference to be assigned? Please be careful not to do this type of operation in the future - there is almost never a good case for doing this.

Reply:
Noted.